subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -641,15 +641,6 @@ uint32_t radio_is_done(void)
 }
 #endif /* !CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER */
 
-uint32_t radio_is_tx_done(void)
```

Comment:
It would be nice if this revert commit elaborated on why it is doing the revert, specially as the next commit reverts the revert :)